CMakePresets.json

```diff
@@ -0,0 +1,59 @@
+{
+    "version": 3,
```

Comment:
Could you perhaps add a schema reference as well, so that auto-completion work? See https://martin-fieber.de/blog/cmake-presets/#create-a-preset for an example.